tools/idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/webimport/lucide/data/LucideRepository.kt

```
@@ -0,0 +1,98 @@
+                    element = svgElement,
+                    attributeName = ATTR_STROKE,
+                    currentValue = DEFAULT_STROKE_COLOR,
+                    newValue = settings.color.toHexString(),
+                )
+            }
+        }
+    }
+
+    private fun Color.toHexString(): String {
+        val argb = this.toArgb()
+        return String.format("#%06X", 0xFFFFFF and argb)
```

detekt (1.23.8)

[warning] String.format("#%06X", 0xFFFFFF and argb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)